src/client/index.js

```diff
@@ -1,15 +1,19 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
-import * as serviceWorker from "./serviceWorker";
-import { BrowserRouter as Router } from "react-router-dom";
-
-ReactDOM.render(
-  <Router>
-    <App />
-  </Router>,
-  document.getElementById("root")
-);
-
-serviceWorker.unregister();
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import { BrowserRouter as Router } from "react-router-dom";
+import MealsProvider from "./contexts/MealsContext";
+import "./index.scss";
+
+ReactDOM.render(
+  <Router>
+    <MealsProvider>

```

Comment:
so it will not be the best practice to wrap the app component by MealsProvider down here 
the best place is at app component